sklearn/metrics/cluster/tests/test_unsupervised.py

```diff
@@ -22,7 +22,6 @@ def test_silhouette():
     X_lil = sp.lil_matrix(X_dense)
     y = dataset.target
 
-    X_dense_score = {}
     for X in [X_dense, X_csr, X_dok, X_lil]:
```

Comment:
Only 'csc' and 'csr' are now accepted. I don't understand how dok and lil are expected to work.